### ISSUE TITLE
Unable to import labelbox

### ISSUE THREAD

**Issue by sc268** [Labelbox/labelbox-python#1626]
Hi Team,

Recently I cannot import labelbox on version `3.69.1`, `3.70.0` on python 3.8.15. Here is the code to reproduce:
```
!pip install labelbox
import labelbox as lb
```
And the error log below. 

Thanks! 
```
>>> import labelbox as lb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/labelbox/__init__.py", line 5, in <module>
    from labelbox.client import Client
  File "/opt/conda/lib/python3.8/site-packages/labelbox/client.py", line 26, in <module>
    from labelbox.schema import role
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/__init__.py", line 14, in <module>
    import labelbox.schema.project
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/project.py", line 45, in <module>
    from labelbox.data.serialization import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/__init__.py", line 1, in <module>
    from .labelbox_v1 import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/__init__.py", line 1, in <module>
    from .converter import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/converter.py", line 1, in <module>
    from labelbox.data.serialization.labelbox_v1.objects import LBV1Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/objects.py", line 10, in <module>
    from .classification import LBV1Checklist, LBV1Classifications, LBV1Radio, LBV1Text, LBV1Dropdown
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/classification.py", line 5, in <module>
    from .feature import LBV1Feature
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/feature.py", line 6, in <module>
    from ...annotation_types.types import Cuid
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/__init__.py", line 1, in <module>
    from .geometry import Line
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/__init__.py", line 3, in <module>
    from .mask import Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/mask.py", line 8, in <module>
    from ..data import MaskData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/__init__.py", line 1, in <module>
    from .audio import AudioData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/audio.py", line 6, in <module>
    class AudioData(BaseData, _NoCoercionMixin):
  File "pydantic/main.py", line 197, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "/opt/conda/lib/python3.8/typing.py", line 774, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class
>>> import labelbox as lb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/labelbox/__init__.py", line 5, in <module>
    from labelbox.client import Client
  File "/opt/conda/lib/python3.8/site-packages/labelbox/client.py", line 26, in <module>
    from labelbox.schema import role
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/__init__.py", line 14, in <module>
    import labelbox.schema.project
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/project.py", line 45, in <module>
    from labelbox.data.serialization import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/__init__.py", line 1, in <module>
    from .labelbox_v1 import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/__init__.py", line 1, in <module>
    from .converter import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/converter.py", line 1, in <module>
    from labelbox.data.serialization.labelbox_v1.objects import LBV1Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/objects.py", line 10, in <module>
    from .classification import LBV1Checklist, LBV1Classifications, LBV1Radio, LBV1Text, LBV1Dropdown
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/classification.py", line 5, in <module>
    from .feature import LBV1Feature
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/feature.py", line 6, in <module>
    from ...annotation_types.types import Cuid
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/__init__.py", line 1, in <module>
    from .geometry import Line
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/__init__.py", line 3, in <module>
    from .mask import Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/mask.py", line 8, in <module>
    from ..data import MaskData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/__init__.py", line 1, in <module>
    from .audio import AudioData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/audio.py", line 6, in <module>
    class AudioData(BaseData, _NoCoercionMixin):
  File "pydantic/main.py", line 197, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "/opt/conda/lib/python3.8/typing.py", line 774, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class
```

**Comment by Gabefire** [comment]
Hey @sc268 
Ill take a peak and get back to you.

**Comment by Gabefire** [comment]
Hey,
I tested this locally on a fresh install with this code:
```python
import labelbox as lb
import platform
print(platform.python_version())
print(lb.__version__)
```

and it worked fine
Output: 
3.8.15
3.69.1

Are you able to download the latest minor release of 3.8 to test if that fixes your issue

**Comment by sc268** [comment]
Thanks @Gabefire somehow I got the same error
```
>>> import platform
>>> print(platform.python_version())
3.8.15
>>> import labelbox 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/labelbox/__init__.py", line 5, in <module>
    from labelbox.client import Client
  File "/opt/conda/lib/python3.8/site-packages/labelbox/client.py", line 26, in <module>
    from labelbox.schema import role
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/__init__.py", line 14, in <module>
    import labelbox.schema.project
  File "/opt/conda/lib/python3.8/site-packages/labelbox/schema/project.py", line 45, in <module>
    from labelbox.data.serialization import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/__init__.py", line 1, in <module>
    from .labelbox_v1 import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/__init__.py", line 1, in <module>
    from .converter import LBV1Converter
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/converter.py", line 1, in <module>
    from labelbox.data.serialization.labelbox_v1.objects import LBV1Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/objects.py", line 10, in <module>
    from .classification import LBV1Checklist, LBV1Classifications, LBV1Radio, LBV1Text, LBV1Dropdown
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/classification.py", line 5, in <module>
    from .feature import LBV1Feature
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/serialization/labelbox_v1/feature.py", line 6, in <module>
    from ...annotation_types.types import Cuid
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/__init__.py", line 1, in <module>
    from .geometry import Line
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/__init__.py", line 3, in <module>
    from .mask import Mask
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/geometry/mask.py", line 8, in <module>
    from ..data import MaskData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/__init__.py", line 1, in <module>
    from .audio import AudioData
  File "/opt/conda/lib/python3.8/site-packages/labelbox/data/annotation_types/data/audio.py", line 6, in <module>
    class AudioData(BaseData, _NoCoercionMixin):
  File "pydantic/main.py", line 197, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "/opt/conda/lib/python3.8/typing.py", line 774, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class
```

**Comment by Gabefire** [comment]
I meant to upgrade your Python version to 3.8.18, the latest patch for 3.8
That issubclass() function comes from a python library https://github.com/python/typeshed

**Comment by sc268** [comment]
Confirmed the issue is resolved with python 3.8.18.  Thanks @Gabefire !

**Comment by Gabefire** [comment]
@sc268 No problem! Thank you for reaching out!l
